### PR TITLE
fix: remove unnecessary characters in Community Inspirations 

### PR DESCRIPTION
**Pull request by ykmsd**
Removed unnecessary `[` and `"` 📝

<a href="https://stackblitz.com/~/github.com/stackblitz/webcontainer-docs/tree/web-publisher/ykmsd/community-inspirations">Review PR in StackBlitz Codeflow</a> _Submitted with [StackBlitz Web Publisher](https://developer.stackblitz.com/codeflow/integrating-web-publisher)._

### CODE REVIEW

File: docs/guides/community-inspirations.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ head:
 This page is a tribute to our Community who have been using WebContainers over the past two years - and who have been building with WebContainer API since it was released in June 2022 in a private alpha. Here are some of the projects that are powered by it.
 
 ::: tip Get your project featured!
-Have a project you'd like us to showcase here and on our social media? [Reach out to our [Developer Advocates on Discord](https://github.com/stackblitz/webcontainer-docs/issues/new?assignees=sylwiavargas&labels=documentation&template=%E2%9C%A8-feature-your-project.md&title=%5B%E2%9C%A8+Project+submission%5D)!
+Have a project you'd like us to showcase here and on our social media? Reach out to our [Developer Advocates on Discord](https://github.com/stackblitz/webcontainer-docs/issues/new?assignees=sylwiavargas&labels=documentation&template=%E2%9C%A8-feature-your-project.md&title=%5B%E2%9C%A8+Project+submission%5D)!
```
Comment (ykmsd) [inline docs/guides/community-inspirations.md:17]: Actually is this link correct? This is not on Discord but GitHub? 🤔

**Review by sylwiavargas — APPROVED** [review]
Thank you

File: docs/guides/community-inspirations.md (hunk 1 above)
Comment (sylwiavargas) [reply docs/guides/community-inspirations.md:17]:
```suggestion
Have a project you'd like us to showcase here and on our social media? Reach out to our [Developer Advocates](https://github.com/stackblitz/webcontainer-docs/issues/new?assignees=sylwiavargas&labels=documentation&template=%E2%9C%A8-feature-your-project.md&title=%5B%E2%9C%A8+Project+submission%5D)!
```

File: docs/guides/community-inspirations.md [hunk 2]
```diff
@@ -70,5 +70,5 @@ Tech Educator's Toolkit is an early exploration of WebContainer API and of a new
 [![A still from Dan Jutan's talk](/img/community/dan_jutan_talk.png)](https://www.youtube.com/watch?v=R-1y3Ti3ng4)
 
 ::: tip Get your project featured!
-Have a project you'd like us to showcase here and on our social media? [Reach out to our [Developer Advocates on Discord](https://github.com/stackblitz/webcontainer-docs/issues/new?assignees=sylwiavargas&labels=documentation&template=%E2%9C%A8-feature-your-project.md&title=%5B%E2%9C%A8+Project+submission%5D)!
+Have a project you'd like us to showcase here and on our social media? Reach out to our [Developer Advocates on Discord](https://github.com/stackblitz/webcontainer-docs/issues/new?assignees=sylwiavargas&labels=documentation&template=%E2%9C%A8-feature-your-project.md&title=%5B%E2%9C%A8+Project+submission%5D)!
```
Comment (ykmsd) [inline docs/guides/community-inspirations.md:73]:
```suggestion
Have a project you'd like us to showcase here and on our social media? Reach out to our [Developer Advocates](https://github.com/stackblitz/webcontainer-docs/issues/new?assignees=sylwiavargas&labels=documentation&template=%E2%9C%A8-feature-your-project.md&title=%5B%E2%9C%A8+Project+submission%5D)!
```

Comment (sylwiavargas) [reply docs/guides/community-inspirations.md]: perfect

**Pull request merged**